.github/ISSUE_TEMPLATE/1.general_issue_report.yaml

```
@@ -0,0 +1,77 @@
+name: Issue Report
+description: File an issue report
```

```suggestion
name: Bug Report
description: File an bug report
```

Since this template is bug-oriented, I think we should make a second issue template specifically for feature requests.

---

agreed.  issues should probably go to discord, and if it's something that can be documented then we add it to the documentation or a FAQ.